src/lib/pages/account-details/components/nfts/NftsOverview.tsx

```diff
@@ -54,13 +54,13 @@ export const NftsOverview = ({
         <AccountSectionWrapper title="NFTs" totalData={totalCount}>
           <Flex
             direction="column"
-            borderBottom={data?.nfts?.length ? "1px solid" : "0px"}
+            borderBottom={data?.items?.length ? "1px solid" : "0px"}
             borderBottomColor="gray.700"
-            mb={data?.nfts?.length ?? 12}
-            pb={data?.nfts?.length ?? 8}
+            mb={data?.items?.length ?? 12}
+            pb={data?.items?.length ?? 8}
```

Comment:
data?.items.length

Reply:
Also, it's strange that we're using item length as both mb and pb.